src/Raven.Studio/typescript/components/pages/database/documents/allRevisions/partials/AllRevisionsAboutView.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { AboutViewFloating, AccordionItemWrapper } from "components/common/AboutView";
+import { useAppUrls } from "hooks/useAppUrls";
+import { useAppSelector } from "components/store";
+import { databaseSelectors } from "components/common/shell/databaseSliceSelectors";
+
+export default function AllRevisionsAboutView() {
+    const { appUrl } = useAppUrls();
+    const activeDatabaseName = useAppSelector(databaseSelectors.activeDatabaseName);
+
+    return (
+        <AboutViewFloating>
+            <AccordionItemWrapper
+                targetId="1"
+                icon="about"
+                color="info"
+                description="Get additional info on this feature"
+                heading="About this view"
+            >
+                <p>
+                    This view displays all revisions for every document in the database.
+                    <br />
+                    You can filter the revisions by their <strong>Collection</strong> or <strong>Type</strong>:
+                    <p className={"mt-2"}>
```

Comment:
Let's change this because we added a rule to eslint that highlights this as a warning
```suggestion
                    <p className="mt-2">
```

Reply:
ok, thanks, good to know